core-api/src/main/java/org/silverpeas/core/date/period/DayPeriod.java

```diff
@@ -28,7 +28,9 @@
 /**
  * User: Yohann Chastagnier
  * Date: 21/04/13
+ * @deprecated Use {@link org.silverpeas.core.date.Period}
  */
+@Deprecated
 public class DayPeriod extends Period {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)